README.md

```diff
@@ -2,8 +2,10 @@
 NullableArrays.jl
 =================
 [![Build Status](https://travis-ci.org/JuliaStats/NullableArrays.jl.svg?branch=master)](https://travis-ci.org/JuliaStats/NullableArrays.jl)
+[![Coverage Status](https://coveralls.io/repos/github/JuliaStats/NullableArrays.jl/badge.svg?branch=master)](https://coveralls.io/github/JuliaStats/NullableArrays.jl?branch=master)
```

Comment:
Could you call this "Coveralls" or something to distinguish it from "Codecov"? Could be worth fixing in PkgDev itself.